src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -533,6 +533,104 @@ function transformTopLevelAtom(atom, field) {
   }
 }
 
+function relativeTimeToDate(text, now = new Date()) {
+  let parts = text.split(' ');
+  if (!parts.length) {
+    return { status: 'error', info: 'Not a time string' };
+  }
+
+  // Filter out whitespace
+  parts = parts.filter((part) => part !== '');
+
+  const future = parts[0] === 'in';
+  const past = parts[parts.length - 1] === 'ago';
```

Comment:
Hmm, wondering if these should be case insensitive. We could lowercase the entire string in advance before processing.

Reply:
lowercased on https://github.com/parse-community/parse-server/pull/4289/files#diff-82f0219ca721372e8f2bdbc45c83bbdcR537

Reply:
Cool